locustempus/main/forms.py

```diff
@@ -1,9 +1,32 @@
+from courseaffils.models import Course
 from django import forms
 from django.conf import settings
 from django.forms.formsets import formset_factory
 from django.core.validators import RegexValidator
 
 
+class CourseForm(forms.ModelForm):
+    description = forms.CharField(widget=forms.Textarea, required=False)
+
+    def save(self, commit=True):
+        course = super().save(commit=False)
+        description = self.cleaned_data.get('description', None)
+        if description:
```

Comment:
This test disallows the ability to clear out the description. A better test would be `if description is not None`. I'll make the change.